CONTRIBUTING.md

```diff
@@ -19,7 +19,6 @@ We use [Lerna](https://lernajs.io/) to manage the packages in this repository. Y
 
 1. Clone the repository
 1. Install dependencies using `npm install` in the root of the project
-1. Run `npm run build` in the root of the project once.
```

Comment:
True, but the step below specifies to run `npm test` on the root **or in individual packages**. In that last case, you do need to build it once before. I think that deserves an explanation if we're removing that line.

Reply:
Ok, I'll add info that before running `npm run test:stryker` you have to run `npm run build`